src/cockpit/bridge.py

```diff
@@ -154,6 +154,26 @@ def try_to_receive_stderr():
             os.close(stderr_fd)
 
 
+def setup_logging(debug: bool):
+    """Setup our logger with optional filtering of modules if COCKPIT_DEBUG env is set"""
+
+    cfg = {}
+    modules = set(os.getenv('COCKPIT_DEBUG', '').split(','))
+
+    def filter_record(record: logging.LogRecord):
+        if record.name in modules or 'all' in modules:
+            return record
+
+    if debug or modules:
+        cfg['level'] = logging.DEBUG
+    if modules:
+        handler = logging.StreamHandler()
+        handler.addFilter(filter_record)
+        cfg['handlers'] = [handler]
+
+    logging.basicConfig(**cfg)
```

Comment:
I'm trying not to hardcode our defaults, but just realised we can always initialise logging.StreamHandler() imo.